src/react/actions/s3object.ts

```diff
@@ -198,11 +198,11 @@ export function createFolder(
   };
 }
 
-function _getObjectListWithSignedUrlAndLockStatus<
+export function _getObjectListWithSignedUrlAndLockStatus<
```

Comment:
Nit: if we make it a public function we should remove the `_` prefixing the name
```suggestion
export function getObjectListWithSignedUrlAndLockStatus<
```